devServer.js

```diff
@@ -13,7 +13,7 @@ app.use(require('webpack-dev-middleware')(compiler, {
 
 app.use(require('webpack-hot-middleware')(compiler));
 
-app.get('/', function(req, res) {
```

Comment:
Jason made this change.. but it's working for me without merging this in right now..